darts/models/forecasting/moving_average.py

```diff
@@ -0,0 +1,35 @@
+"""
+Moving Average
+-------------------------------
+"""
+import numpy as np
+
+from darts.models.forecasting.forecasting_model import ForecastingModel
+from darts.timeseries import TimeSeries
+
+
+class MovingAverageForecaster(ForecastingModel):
+    def __init__(self, window: int):
+        """Moving Average Model
+
+        Parameters
+        ----------
+        window
+            The length of the window over which to average values.
+        """
+        super().__init__()
+        self.window = window
+        self.mean_val = None
+
+    def __str__(self):
+        return "Moving average predictor model"
+
+    def fit(self, series: TimeSeries):
+        super().fit(series)
+        self.mean_val = series[-self.window :].univariate_values().mean()
```

Comment:
Yes, this should be an easy extension. If I understand the library structure correctly, to be able to accept multivariate series input this also requires extending from `ForecastingModel` to `GlobalForecastingModel`.

This would change the input argument to `predict` a bit, specifically `series` can now be in the input. So we would have two cases:
- `series` is None: Use the values saved from `fit` to do the prediction.
- if `series` is in the input: ignore the values/means from the fit and directly use the new `series` instead.

Reply:
No it wouldn't have to inherit from `GlobalForecastingModel`. Global models are those that support fitting on multiple series - here we are only talking about fitting on (single) multivariate series.

Reply:
If I inherit from `ForecastingModel` with a simple time series with 2 components then I get an error:
```
ERROR: AssertionError: Only univariate TimeSeries instances support this method
```

Full code:
```
ts = TimeSeries.from_dataframe(pd.DataFrame({'a': np.arange(10), 'b': np.arange(10) + 5}))
ma = MovingAverageForecaster(window=5)
ma.fit(ts)
```